html_reference.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang='en'>
+	<head>
+		<link rel="stylesheet" href="css_stylesheet.css">
+		<script src="https://kit.fontawesome.com/c90803ff49.js" crossorigin="anonymous"></script>
+		<title>Milo Girardet</title>
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+	</head>
+	<body onTouchStart="">
+	<header class="contactDetails">
+		<div class="container">
+			<div class="buttonsSentLeft">
+				<ul class="contactAnchors">
+					<li><a href="#" class="contactAnchorsStyling">milogirardet@gmail.com</a></li>
+					<li><a href="#" class="contactAnchorsStyling">07519645711</a></li>
+				</ul>
+				<!-- inconsistent class naming is used due to fab being links to specific logos from an external website -->
+				<div class="socialMediaButtons">
+					<a><i class="fab fa-linkedin"></i></a>
+					<a><i class="fab fab fa-github-square"></i></a>
+					<a><i class="fas fa-share-alt-square"></i></a>
+				</div>
+			</div>
+			<h1 class="myName">milo_girardet  <i class="fas fa-user-astronaut" id="astronaut"></i></h1>					
+		</div>
+	</header>
+		<div class="paragraphs">
+			<div class="aboutMePosition">
+				<h2 class="container">full_stack_developer  <i class="fas fa-code" id="code"></i></h2>
```

Comment:
Do these h2s need the container class? Seems to work without

Reply:
I think it was a bit of stray code from previous version.